database/src/main/scala/cromwell/database/obj/Metadatum.scala

```diff
@@ -10,6 +10,7 @@ case class Metadatum
   index: Option[Int],
   attempt: Option[Int],
   value: Option[String],
+  valueType: Option[String],
```

Comment:
Ideally value would be an ADT with an associated `TypedType` which would allow us to convert into two or more database columns...

Ok never mind.